index.js

```diff
@@ -92,6 +97,38 @@ function isFormMethodDialog(el) {
   return el.getAttribute('method').toLowerCase() === 'dialog';
 }
 
+/**
+ * @param {!DocumentFragment|!Element} hostElement
+ * @return {?Element} 
+ */
+function findFocusableElementInShadowDom(hostElement) {
```

Comment:
This isn't really "in Shadow DOM", because it's just in ... something. If we recurse, then it's inside Shadow DOM.

So maybe "findFocusableElementWithin".

Reply:
Good catch - changed.